sheepdog/utils/__init__.py

```diff
@@ -164,8 +154,7 @@ def check_and_call(program, project, *args, **kwargs):
             )
             if not project_node:
                 raise NotFoundError('Project {} not found'.format(project))
-            flask.g.dbgap_accession_numbers = "{},{}".format(program_node.dbgap_accession_number,
-                                                             project_node.dbgap_accession_number)
+            phsids = [program_node.dbgap_accession_number, project_node.dbgap_accession_number]
```

Comment:
variable `phsids` isn't assigned. I checked the master branch and it also wasn;t assigned there.